syscall_stub.go

```diff
@@ -0,0 +1,48 @@
+//go:build !windows
+// +build !windows
+
+package comutil
+
+import (
+	"errors"
+	"unsafe"
+
+	"google3/third_party/golang/googleuuid/uuid"
```

Comment:
It looks like the imports have inadvertently been changed here.

Reply:
Yes indeed. Sorry about that. Should be fixed.